### DISCUSSION TITLE
Strange artifacts when upscaling with Tiled VAE

### DISCUSSION THREAD

**Discussion by iSupremacy** [pkuliyi2015/multidiffusion-upscaler-for-automatic1111#397] [General]
https://imgur.com/a/s1VOkev
Tried multiple upscalers and samplers. sometimes it happens sometimes it doesn't. I think it doesn't happen with Tiled VAE OFF, but renders are impossible with it off.

appreciate any assistance.

**Comment by Waaaaaaaaaaaaaaaaaaaaaaaaa** [comment]
Though I can't see your image (404) your problem could be similar to what mine was. Try disabling "Fast Decoder" as it seems to cause artifacts with some VAEs. I noticed issues with SDXL in particular.

Those two messages are just stating that on certain steps the extensions were not used as they were unnecessary.

**Reply by iSupremacy** [reply]
> Though I can't see your image (404) your problem could be similar to what mine was. Try disabling "Fast Decoder" as it seems to cause artifacts with some VAEs. I noticed issues with SDXL in particular.
> 
> Those two messages are just stating that on certain steps the extensions were not used as they were unnecessary.

For got to update my post, yep i found out it was fast decoder. Its sad we lose out on that speed using it.
